test/basic-querying.test.js

```diff
@@ -26,6 +26,17 @@ describe('basic-querying', function() {
 
   });
 
+  describe('forceId', function() {
+    it('returns error when default forceId is true', function(done) {
```

Comment:
This test name does not make sense to me. How about "it defaults to forceId:true for generated id property"?

Also "basic-querying.test.js" looks like a wrong place for a test that creates new model instances. I think the "create" block in manipulation.test.js would be a better place - see https://github.com/strongloop/loopback-datasource-juggler/blob/master/test/manipulation.test.js#L58